Bottom: 417fc4c88c560a3593fa85ddad1fec150f915b35
Top:    846f25c3c1b3601d504e618680abfea3b2ad164a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 00:28:56 +0000

Refresh of remove-remaining-rcs-id-identi

---

diff --git a/src/hid/bom/bom.c b/src/hid/bom/bom.c
index 1835439..e67450a 100644
--- a/src/hid/bom/bom.c
+++ b/src/hid/bom/bom.c
@@ -333,6 +333,8 @@ PrintBOM (void)
     const char *fmt = "%c UTC";
     strftime (utcTime, sizeof (utcTime), fmt, gmtime (&currenttime));
   }
+  fprintf (fp, "# $Id");
+  fprintf (fp, "$\n");
   fprintf (fp, "# PcbXY Version 1.0\n");
   fprintf (fp, "# Date: %s\n", utcTime);
   fprintf (fp, "# Author: %s\n", pcb_author ());
@@ -498,6 +500,8 @@ PrintBOM (void)
       return 1;
     }
 
+  fprintf (fp, "# $Id");
+  fprintf (fp, "$\n");
   fprintf (fp, "# PcbBOM Version 1.0\n");
   fprintf (fp, "# Date: %s\n", utcTime);
   fprintf (fp, "# Author: %s\n", pcb_author ());
diff --git a/tests/golden/hid_bom1/bom_general.bom b/tests/golden/hid_bom1/bom_general.bom
index ee15f7b..e069ea1 100644
--- a/tests/golden/hid_bom1/bom_general.bom
+++ b/tests/golden/hid_bom1/bom_general.bom
@@ -1,3 +1,4 @@
+# $Id$
 # PcbBOM Version 1.0
 # Date: Wed Jun 24 21:42:21 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom1/bom_general.xy b/tests/golden/hid_bom1/bom_general.xy
index c391ac2..dfcd7de 100644
--- a/tests/golden/hid_bom1/bom_general.xy
+++ b/tests/golden/hid_bom1/bom_general.xy
@@ -1,3 +1,4 @@
+# $Id$
 # PcbXY Version 1.0
 # Date: Wed Jun 24 21:42:21 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom2/bom_general.xy b/tests/golden/hid_bom2/bom_general.xy
index 56bf534..c2c9ea2 100644
--- a/tests/golden/hid_bom2/bom_general.xy
+++ b/tests/golden/hid_bom2/bom_general.xy
@@ -1,3 +1,4 @@
+# $Id$
 # PcbXY Version 1.0
 # Date: Wed Jun 24 21:42:22 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom2/test.bom b/tests/golden/hid_bom2/test.bom
index 895ee5b..c2677f5 100644
--- a/tests/golden/hid_bom2/test.bom
+++ b/tests/golden/hid_bom2/test.bom
@@ -1,3 +1,4 @@
+# $Id$
 # PcbBOM Version 1.0
 # Date: Wed Jun 24 21:42:22 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom3/bom_general.bom b/tests/golden/hid_bom3/bom_general.bom
index 895ee5b..c2677f5 100644
--- a/tests/golden/hid_bom3/bom_general.bom
+++ b/tests/golden/hid_bom3/bom_general.bom
@@ -1,3 +1,4 @@
+# $Id$
 # PcbBOM Version 1.0
 # Date: Wed Jun 24 21:42:22 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom3/test.xy b/tests/golden/hid_bom3/test.xy
index 56bf534..c2c9ea2 100644
--- a/tests/golden/hid_bom3/test.xy
+++ b/tests/golden/hid_bom3/test.xy
@@ -1,3 +1,4 @@
+# $Id$
 # PcbXY Version 1.0
 # Date: Wed Jun 24 21:42:22 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom4/bom_general.bom b/tests/golden/hid_bom4/bom_general.bom
index b3cc77f..bb246f5 100644
--- a/tests/golden/hid_bom4/bom_general.bom
+++ b/tests/golden/hid_bom4/bom_general.bom
@@ -1,3 +1,4 @@
+# $Id$
 # PcbBOM Version 1.0
 # Date: Wed Jun 24 21:42:23 2009 UTC
 # Author: Dan McMahill
diff --git a/tests/golden/hid_bom4/bom_general.xy b/tests/golden/hid_bom4/bom_general.xy
index 2933983..fb1350e 100644
--- a/tests/golden/hid_bom4/bom_general.xy
+++ b/tests/golden/hid_bom4/bom_general.xy
@@ -1,3 +1,4 @@
+# $Id$
 # PcbXY Version 1.0
 # Date: Wed Jun 24 21:42:23 2009 UTC
 # Author: Dan McMahill
